originals/h.yaml

```
@@ -222,6 +222,24 @@
     genres:
     - Real-Time Strategy
 
+- name: 'Higurashi: When They Cry Hou'
```

```suggestion
- name: 'Higurashi: When They Cry'
```
I think it's better to refer to the original game here https://en.wikipedia.org/wiki/Higurashi_When_They_Cry

---

The clones added here are specifically for the console remake. The original games are already source-available, though they are missing all of the extra features the remake offers, such as voice acting, visual improvements, etc.